pkg/webhook/policy.go

```diff
@@ -181,24 +180,34 @@ func (h *validationHandler) Handle(ctx context.Context, req admission.Request) a
 	}
 
 	res := resp.Results()
-	msgs := h.getDenyMessages(res, req)
+	msgs, enforcementAction := h.getDenyMessages(res, req)
```

Comment:
We should return warning messages and violation messages as two different variables.